tests/integration/suite/test_istio.py

```diff
@@ -66,6 +66,45 @@ def test_destination_rule(admin_pc):
     client.delete(ns)
 
 
+@pytest.mark.nonparallel
+def test_destination_rule_on_cookie(admin_pc):
+    client = admin_pc.client
+    ns = admin_pc.cluster.client.create_namespace(
+        name=random_str(),
+        projectId=admin_pc.project.id)
+    name = random_str()
+    cookie_name = name + "_cookie"
+    client.create_destinationRule(
+        name=name,
+        namespaceId=ns.id,
+        host="test",
+        subsets=[{
+            "name": "v1",
+            "labels": {
+                "version": "v1",
+            }
+        }],
+        trafficPolicy={
+            "loadBalancer": {
+                "consistentHash": {
+                    "httpCookie": {
+                        "ttl": "0s",
+                        "name": cookie_name,
+                    }
+                }
+            }
+        }
+    ),
+    destinationRules = client.list_destinationRule(
+        namespaceId=ns.id
+    )
+    assert len(destinationRules) == 1
+    assert destinationRules.data[0].trafficPolicy.loadBalancer\
+        .consistentHash.httpCookie.name == cookie_name
+    client.delete(destinationRules.data[0])
+    client.delete(ns)
```

Comment:
Should use remove_resource. In current state, if something fails they won't be deleted.